githubAPI/octokit.js

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require("@octokit/rest");
+require('dotenv').config();
+
+const octokit = new Octokit({
+  auth: process.env.WEBHOOK_SECRET,
+  log: {
+    debug: () => {},
+    info: () => {},
+    warn: console.warn,
+    error: console.error
+  }
+});
+
+if(!octokit.auth) {
+  console.log('Missing authentication token. Please add correct environment variable to file .env!');
+  return undefined;
```

Comment:
I think good practice would be log a warning, but return the `octokit` object as is? IMO better to defer to the caller for validation; passes the responsibility down to the caller but also provides more flexibility to how the caller might want to use the object.

Reply:
Indeed. I'll make this change with in a single commit within the next suggested changes.